lib/models/task.dart

```diff
@@ -11,62 +10,41 @@ class Task extends Undertaking {
   /// Default constructor with minimum required info
   /// Good for if you want to add a new task from user with missing fields
   Task(
-      {String? name,
-      String? id,
-      String? description,
+      {super.name,
+      super.id,
+      super.description,
       bool completed = false,
-      String? location,
-      String? color,
-      List<String>? tags,
-      Recurrence? recurrenceRules,
-      DateTime? timeStart,
+      super.location,
+      super.color,
+      super.tags,
+      super.recurrenceRules,
+      super.timeStart,
       DateTime? timeDue,
       DateTime? timeCurrent,
-      DateTime? timeCreated,
-      DateTime? timeModified})
-      : super(
-            name: name,
-            id: id,
-            description: description,
-            location: location,
-            color: color,
-            tags: tags,
-            recurrenceRules: recurrenceRules,
-            timeStart: timeStart,
-            timeCreated: timeCreated,
-            timeModified: timeModified) {
+      super.timeCreated,
+      super.timeModified}) {
```

Comment:
I'm not super familiar with the syntax but this means the same thing as putting it in the constructor right? Or idk if it's like setting the variable directly vs using a setter. If it's using a setter then it might overwrite the timeModified variable with the current time

Reply:
same thing for event *

Reply:
If it passes the task tests then it should be good

Reply:
this is what `dart fix` suggested to do, but I can run the task tests and see if that passes

Reply:
seems like it passes locally on my machine